### DISCUSSION TITLE
Generating SSL Certificate issue

### DISCUSSION THREAD

**Discussion by parasop** [vercel/community#2922] [Help] [answered]
### Summary

I added my custom domain to vercel. It's more than 1 hour it still generating ssl certificate

### Steps to Reproduce

like it's keep loading  to generate certificate

**Comment by smaeda-ks** [comment]
@parasop What's the domain name?

**Reply by parasop** [reply]
https://competitivematch.it

**Reply by smaeda-ks** [reply]
@parasop Could you disable DNSSEC in your domain registrar? We don't support it as of today.
https://dnsviz.net/d/competitivematch.it/dnssec/

**Reply by parasop** [reply]
I purchased from register.it and I don't see there option to disable this

**Reply by smaeda-ks** [reply]
https://www.register.it/help/come-abilitare-il-protocollo-dnssec-sul-dominio/

**Reply by parasop** [reply]
but i am using cloudflare so i am not able to edit that

**Reply by smaeda-ks** [reply, marked as answer]
@parasop you may reach out to register.it for further assistance. You should probably have disabled DNSSEC before switching to Cloudflare nameservers:
https://developers.cloudflare.com/dns/dnssec/#disable-dnssec
